### PR TITLE
New QA device images

### PR DESCRIPTION
**Pull request by queroautomacao**

### CODE REVIEW

File: docs/devices/SNZB-05P.md [hunk 1]
```diff
@@ -1,76 +0,0 @@
----
-title: "SONOFF SNZB-05P control via MQTT"
-description: "Integrate your SONOFF SNZB-05P via Zigbee2MQTT with whatever smart home infrastructure you are using without the vendor's bridge or gateway."
```
Comment (Koenkk) [inline docs/devices/SNZB-05P.md:3]: why was this file modified?

Comment (queroautomacao) [reply docs/devices/SNZB-05P.md:3]: No, only added QA devices images.

Comment (Koenkk) [reply docs/devices/SNZB-05P.md:3]: Thats not the case, look at the changes in the pr
